cmd/sonobuoy/app/operations/cp.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operations
+
+import (
+// TODO
+)
+
+func CopyResults(path string) error {
+	// pretty straight forwards but will
+	return nil
```

Comment:
might be better if the stubs returned `errors.New("not  implemented")`

Reply:
+1 - Done.